galsim/base.py

```diff
@@ -1131,6 +1171,8 @@ def drawImage(self, image=None, nx=None, ny=None, bounds=None, scale=None, wcs=N
                             numpy.float32]
         @param method       Which method to use for rendering the image.  See discussion above
                             for the various options and what they do. [default: 'auto']
+        @param area         Collecting area of telescope in cm^2.  [default: 1.]
```

Comment:
I think it's alright as is.  In the long run, I think using the astropy.units framework for more things looks nice (especially the various `scale_unit` kwargs), but I don't think we need to tackle this everywhere right now.